<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUILinkedList</class>
 <widget class="QMainWindow" name="GUILinkedList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUILinkedList</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>78</width>
      <height>99</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select Type: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="StringList">
       <property name="text">
        <string>String</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="DoubleList">
       <property name="text">
        <string>Double</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="IntList">
       <property name="text">
        <string>Int</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>121</width>
      <height>205</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pop_frontButton">
       <property name="text">
        <string>Pop Front</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_dupeButton">
       <property name="text">
        <string>Remove Duplicates</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyListButton">
       <property name="text">
        <string>Copy List</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectSortButton">
       <property name="text">
        <string>Select Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearList">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>111</width>
      <height>164</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="textLine"/>
     </item>
     <item>
      <widget class="QPushButton" name="push_backButton">
       <property name="text">
        <string>Push Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_frontButton">
       <property name="text">
        <string>Push Front</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="insert_sortedButton">
       <property name="text">
        <string>Insert Sorted</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>17</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Andrew Moss</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CS 003A</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>350</y>
      <width>111</width>
      <height>164</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QLineEdit" name="textLine_2"/>
     </item>
     <item>
      <widget class="QPushButton" name="push_backButton_2">
       <property name="text">
        <string>Push Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_frontButton_2">
       <property name="text">
        <string>Push Front</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="insert_sortedButton_2">
       <property name="text">
        <string>Insert Sorted</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>40</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="graphicsDisplayButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>240</y>
      <width>256</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Display Graphics</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>620</y>
      <width>181</width>
      <height>49</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="portionBegin_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>End</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="portionEnd_2"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>260</width>
      <height>243</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Original List:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="displayButton">
         <property name="text">
          <string>Display List</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="displayPortionButton">
         <property name="text">
          <string>Display Portion</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>330</y>
      <width>258</width>
      <height>283</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Copied List:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_2"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="displayCopy_2">
         <property name="text">
          <string>Display Copy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="displayPortionButton_2">
         <property name="text">
          <string>Display Portion</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>360</y>
      <width>119</width>
      <height>135</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QPushButton" name="pop_frontButton_2">
       <property name="text">
        <string>Pop Front</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_dupeButton_2">
       <property name="text">
        <string>Remove Duplicates</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectSortButton_2">
       <property name="text">
        <string>Select Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearList_2">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>181</width>
      <height>49</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="portionBegin"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>End</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="portionEnd"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget_2</zorder>
   <zorder>layoutWidget_3</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>layoutWidget_4</zorder>
   <zorder>displayButton</zorder>
   <zorder>comboBox</zorder>
   <zorder>portionBegin</zorder>
   <zorder>portionEnd</zorder>
   <zorder>graphicsView</zorder>
   <zorder>graphicsDisplayButton</zorder>
   <zorder>label_7</zorder>
   <zorder>pop_frontButton_2</zorder>
   <zorder></zorder>
   <zorder>layoutWidget_3</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>push_backButton</tabstop>
  <tabstop>insert_sortedButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
